src/labels/CustomLabels.labels

```diff
@@ -4214,6 +4270,548 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
+    </labels>
+    <labels>
+        <fullName>geButtonFormFieldsExpandAll</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button to expand all</shortDescription>
+        <value>Expand All</value>
+    </labels>
+    <labels>
+        <fullName>geButtonFormFieldsModalDeleteSectionAndFields</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button to delete a section and remove its fields</shortDescription>
+        <value>Delete Section and Fields</value>
+    </labels>
+    <labels>
+        <fullName>geButtonSaveAndClose</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Gift Entry Template Builder save button text</shortDescription>
+        <value>Save &amp; Close</value>
+    </labels>
+    <labels>
+        <fullName>geErrorExistingTemplateName</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Field-level error message when template name is not unique</shortDescription>
+        <value>This name has been used by another template. Please enter a unique name.</value>
+    </labels>
+    <labels>
+        <fullName>geErrorFieldPermission</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Field-level error message for insufficient field access</shortDescription>
+        <value>You don't have permission to edit this field.</value>
+    </labels>
+    <labels>
+        <fullName>geErrorPageLevelAdvancedMappingBody</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Page-level error message body when Advanced Mapping off</shortDescription>
+        <value>This feature requires Advanced Mapping. Please enable it in NPSP Settings.</value>
+    </labels>
+    <labels>
+        <fullName>geErrorPageLevelAdvancedMappingHeader</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Page-level error message header when Advanced Mapping off</shortDescription>
+        <value>Enable Advanced Mapping</value>
+    </labels>
+    <labels>
+        <fullName>geErrorPageLevelFieldPermission1</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Page-level error for insufficient field access with fields listed</shortDescription>
+        <!-- {0} is a list. Name of fields e.g. npsp__Amount__c, npsp__Description__c -->
```

Comment:
@psadie-sfdo and @markross-sfdo do you think this variable needs to also show the object the field is on? Something like npe01__OppPayment__c.npsp__Amount__c

Reply:
Though to be honest, I'd almost prefer this to be more human readable if it won't cause confusion or make it hard to find the field. Something like "Payment: Amount"

Reply:
Arrrgh. I just got to the Part 2 of this error message. Were we vague in that error message (NPSP Data Import and target object) because we don't know the object where this field has permission issues?